test-app.gradle

```diff
@@ -20,24 +21,34 @@ ext.applyTestAppSettings = { DefaultSettings defaultSettings ->
 
 ext.applyTestAppModule = { Project project, String packageName ->
     applyNativeModulesAppBuildGradle(project, packageName)
+    
+    def manifestFile = file("$rootDir/app.json")
+    def manifest = new JsonSlurper().parseText(manifestFile.text)
+
+    def resources = manifest["resources"]["android"]    
 
     def generatedAssetsDir = file("${buildDir}/generated/rncli/src/main/assets/")
     generatedAssetsDir.mkdirs()
 
     def generatedResDir = file("${buildDir}/generated/rncli/src/main/res/")
     generatedResDir.mkdirs()
 
-    task copyBundle(type: Copy) {
-        def bundlePath = project.getProperties()
-                .get("testApp.bundle")
+    task copyBundle(type: Copy) {        
+        def bundlePath = resources["bundle"]
 
-        from "${rootDir}/${bundlePath}"
-        into generatedAssetsDir
+        if (bundlePath != null) {
+            from "${rootDir}/${bundlePath}"
+            into generatedAssetsDir
+        }
     }
 
     task copyResources(type: Copy) {
-        def resourcesPath = project.getProperties()
-                .get("testApp.resources")
+        def resourcesPath = resources["resources"]
```

Comment:
Instead of having specific keys for resources and bundles, can we instead infer them by looking at the folder structure? E.g. if the path is `drawable-*/**/*` and `raw/**/*`, we can put them under `res/`, and put everything else under `assets/`? That way, we can use an array like for iOS and allow adding arbitrary assets.

Reply:
Yes, we can do that, but then we will have to handle every android resource directory individually.